client/public/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
+    <meta name="google-signin-client_id" content="558027906323-pp4gp3279cmjjulkggqn77vgklp58vhm.apps.googleusercontent.com">
```

Comment:
Should we be exposing this? Or should this be in a .env file?

Reply:
couldn find a conclusive answer.. still looking